scripts/REFERENCE.md

```
@@ -0,0 +1,142 @@
+```
+  ac-phase                               Interactive menu
+  ac-phase --status                      Show phase progress
+  ac-phase --run                         Run next pending phase
+  ac-phase --run --phase 2               Run specific phase
+  ac-phase --run --all                   Run all remaining phases
+  ac-phase --init                        Regenerate phases from specs
+```
+
+## auto-claude.sh Reference (individual commands)
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)